pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -284,8 +285,9 @@ func ManageClientCABundle(ctx context.Context, lister corev1listers.ConfigMapLis
 	requiredConfigMap, err := resourcesynccontroller.CombineCABundleConfigMaps(
 		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.TargetNamespace, Name: "client-ca"},
 		lister,
-		"kube-apiserver",
-		"",
+		certrotation.AdditionalAnnotations{
+			JiraComponent: "kube-apiserver",
```

Comment:
OK - the second parameter maps to `certrotation.AdditionalAnnotations.Description`.

Reply:
Yep, didn't see a need to set it explicitly to the empty string though, assume that's ok?

Reply:
sure, fine with me. thanks!